packages/console/src/components/Sidebar/icons/BarGraph.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const BarGraph = () => {
+  return (
```

Comment:
should we keep icon as .svg  files and keep them under a static folder?  Like this https://parceljs.org/languages/svg/#importing-as-a-react-component

Reply:
I think in this way, we have more control over the SVG in terms of color rendering, etc.

Reply:
But maybe we should put all the icons in a dedicated folder, like `src/components/icons`?

Reply:
@simeng-li I've tired that approach but didn't work in TypeScript
@demonzoo these icons are for sidebar only i think. Not general icons.

Reply:
OK, but I do think some of these are reusable, like the last two: a circled question mark and document icons

Reply:
@demonzoo they actually will have different strokes / canvas sizes. imo it'll be good if we follow figma libs

Reply:
k let's keep it for now.